components/map/index.js

```diff
@@ -21,26 +29,43 @@ export default class TheMap extends Component {
   }
 
   render() {
+    // const { coordinate } =
```

Comment:
is this commented code needed?

Reply:
the map component isn't related to my US. The component that I developed is WithinBuilding component.

Reply:
this code was added in one of the commits as a part of this PR, maybe it was added by mistake?

Reply:
may be because I based the branch on US2D branch.